src/components/Pokemon.js

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+
+const Pokemon = ({name}) => {
+    const [pokemonInfo, setPokemonInfo] = useState({});
```

Comment:
Ik zou de default waarde aanpassen naar undefined, zodat je een duidelijker verschil hebt tussen of het al wél of niet is ingeladen
```suggestion
    const [pokemonInfo, setPokemonInfo] = useState();
```